src/commands/mod.rs

```diff
@@ -1450,12 +1452,12 @@ fn log_template(settings: &UserSettings) -> String {
                 git_head,
                 commit_id.{prefix_format},
                 if(conflict, label("conflict", "conflict")),
-              )
-              "\n"
-              if(empty, label("empty", "(empty)") " ")
-              if(description, description.first_line(), {DESCRIPTION_PLACEHOLDER_TEMPLATE})
-              "\n"
-            )"#,
+              ),
+              "\n",
+              if(empty, "{{label("empty", "(empty)")}} "),
```

Comment:
Nice to see that quotes inside interpolation work, but it seems hard for a human to parse.

Reply:
yup, infix concatenation operator would be better here.